rpc_util.go

```diff
@@ -209,6 +211,22 @@ func FailFast(failFast bool) CallOption {
 	})
 }
 
+// WithMaxReceiveMessageSize returns a CallOption which sets the maximum message size the client can receive.
+func WithMaxReceiveMessageSize(s int) CallOption {
```

Comment:
Rename this to `MaxReceiveMessageSize`, by convention we don't name CallOption `WithSomething`.